receiver/awscontainerinsightreceiver/receiver_test.go

```diff
@@ -91,14 +90,14 @@ func TestCollectData(t *testing.T) {
 	require.NotNil(t, metricsReceiver)
 
 	r := metricsReceiver.(*awsContainerInsightReceiver)
-	r.Start(context.Background(), nil)
+	require.Error(t, r.Start(context.Background(), nil))
```

Comment:
```suggestion
	require.NoError(t, r.Start(context.Background(), nil))
```

Looking for an error here doesn't seem right.  If it is, please add a comment explaining why.  The same goes for the remaining uses in this file.

Reply:
when `awsContainerInsightReceiver` call `Start`, it will call `newNodeCapacity` function to parse the node capacity through path `/rootfs/proc`, 

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/da2df2664b97cf5e091813c24a3abc11ef77b809/receiver/awscontainerinsightreceiver/internal/host/hostinfo.go#L83

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/da2df2664b97cf5e091813c24a3abc11ef77b809/receiver/awscontainerinsightreceiver/internal/host/nodeCapacity.go#L50

but this path is only valid in container, which is mounted to refer to the host proc directory.

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/da2df2664b97cf5e091813c24a3abc11ef77b809/receiver/awscontainerinsightreceiver/internal/host/utils.go#L26

so when I ran the test on my Mac and Linux machine, they both throw out the error
```
failed to initialize NodeCapacity: lstat /rootfs/proc: no such file or directory
```
I think we should just escape the returned error instead of `require.Error` or `require.NoError`